### ISSUE TITLE
Result of ProtoNet with conv64F on miniImageNet

### ISSUE THREAD

**Issue by huangxilang901** [RL-VIG/LibFewShot#13]
Thanks for sharing the codes and your excellent work.
In Table 2, I see ProtoNet using Conv64F backbone achieved 72% accuracy on miniImageNet 5-way 5-shot setting. However, since ProtoNet did not use data augmentation, my guess is that the possible reason lies in the use of center crop on the meta-test dataset for evalution. Can you explain how you got this result?  
Thanks!

**Comment by huangxilang901** [comment]
> Thanks for sharing the codes and your excellent work. In Table 2, I see ProtoNet using Conv64F backbone achieved 72% accuracy on miniImageNet 5-way 5-shot setting. However, since ProtoNet did not use data augmentation, my guess is that the possible reason lies in the use of center crop on the meta-test dataset for evalution. Can you explain how you got this result? Thanks!

Another question, In Table 1 and 2, I notice that CAN is labeled by Pre-train. However, the original paper did not use pre-trained embedding network. What dose the "Pre-train" mean here?

**Comment by wZuck** [comment]
Hi, we have updated the result of Proto-Conv64F-miniImageNet-Table2 in [Proto](https://github.com/RL-VIG/LibFewShot/tree/main/config/reproduce/Proto), the paper will also be updated, we will release more configs and checkpoints in this folder as soon as possible.
